test/runtest-individual.e2e.js

```diff
@@ -0,0 +1,612 @@
+import { startApp, stopApp, screenshotApp } from './utils'
+import { waitFor } from '@testing-library/dom'
+
+jest.setTimeout(600000)
```

Comment:
Is there a reason behind this number?

Reply:
I just wanted to set the timeout to a large value in case the CI run takes time.